freshmaker/lightblue.py

```diff
@@ -467,6 +467,34 @@ def resolve(self, lb_instance, children=None):
             err = "Cannot resolve the container image: %s" % e
             self.log_error(err)
 
+    def resolve_flatpak_odcs_compose_ids(self):
```

Comment:
There is not much different between this API and `resolve_original_odcs_compose_ids`, could you just add a param to the original API:
```
def resolve_original_odcs_compose_ids(self, from_most_original_build=True):
    ...
    original_nvr = self.nvr
    if from_most_original_build:
        original_nvr =  ArtifactBuild.get_most_original_nvr(self.nvr)
    ...
```

Reply:
Ok. Try to fix it.